www/Gruntfile.js

```diff
@@ -38,8 +38,8 @@ module.exports = function (grunt) {
         files: [
           {
             src: [
-              'less/**/*.less',
-              'node_modules/wikimedia-page-library/build/wikimedia-page-library-transform.css'
+              'node_modules/wikimedia-page-library/build/wikimedia-page-library-transform.css',
```

Comment:
@mdholloway @niedzielski 

Almost forgot - To get the more bullet-proof article theme-ing from `wikimedia-page-library` version `4.5.5` make sure your Android integration patch references `wikimedia-page-library-transform.css` after the upstream CSS.

On iOS the upstream CSS is `styles.css` and `wikimedia-page-library-transform.css` is the first CSS in `styleoverrides.css`:
![screen shot 2017-08-10 at 8 18 57 pm](https://user-images.githubusercontent.com/3143487/29200349-7004a2aa-7e09-11e7-93ce-cf7c309ebb41.png)